code/game/machinery/transformer.dm

```diff
@@ -90,15 +90,15 @@
 	sleep(5)
 
 	use_power(5000) // Use a lot of power.
-	var/mob/living/silicon/robot/R = H.Robotize()
+	var/mob/living/silicon/robot/R = H.Robotize(connect_to_ai = FALSE)
 	R.cell = new /obj/item/stock_parts/cell/upgraded/plus(R, robot_cell_charge)
 
  	// So he can't jump out the gate right away.
 	R.SetLockdown()
 	if(masterAI)
 		R.set_connected_ai(masterAI)
+		R.lawupdate = TRUE
 		R.lawsync()
-		R.lawupdate = 1
```

Comment:
This is a bugfix right? Why was it done?

Reply:
`lawsync` returns early if `lawupdate` is `FALSE`.